propeoplemd/cibox
# Remove Drush dependency from github.yml

- Drush dependency was removed
- ansible.cfg was cleaned
- hosts file was removed
- runner was fixed

How to review:

1. Go to `github` directory.
2. Run `run.sh` and you should get `DEMO` directory with VM configuration.
